lighthouse-core/gather/gather-runner.js

```diff
@@ -552,29 +552,7 @@ class GatherRunner {
     log.time(status);
     const response =
       await passContext.driver.sendCommand('Page.getInstallabilityErrors');
-
-    let errors = response.installabilityErrors;
-    // COMPAT: Before M82, `getInstallabilityErrors` was not localized and just english
```

Comment:
I think we could remove this in a separate PR, wanna do that first?

Reply:
done :) #11782